src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -35,28 +35,24 @@
             <converters:MarginConverter x:Key="MarginConverter" />
         </Grid.Resources>
         <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
             <RowDefinition Height="5" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <TextBlock TextWrapping="Wrap">
-            <Run Text="{x:Static local:GridOptionPreviewControl.CodeStylePageHeader}"/>
-            <Run Text=" "/>
-            <Hyperlink RequestNavigate="LearnMoreHyperlink_RequestNavigate" NavigateUri="{x:Static local:GridOptionPreviewControl.CodeStylePageHeaderLearnMoreUri}">
-                <TextBlock Text="{x:Static local:GridOptionPreviewControl.CodeStylePageHeaderLearnMoreText}"/>
-            </Hyperlink>
-        </TextBlock>
+        <options:CodeStyleNoticeTextBlock Grid.Row="0" Margin="5 0 5 0" />
```

Comment:
📝 Interesting, never though about the commas being optional, but now that I see it, it makes sense because shapes leave out the commas.